docs/contributing.rst

```diff
@@ -0,0 +1,85 @@
+============
+Contributing
+============
+
+We have a strong vision about the scope of this package and the "less is more" principle
```

Comment:
this is duplicated in another part of the docs, I would keep this section and remove the other place (README I think?)